artview/core/common.py

```diff
@@ -155,42 +155,49 @@ def startDisplay(self):
         return self.result, self.independent.isChecked()
 
 ########################
-# Table methods #
+#    Table methods     #
 ########################
 
 
 class CreateTable(QtGui.QTableWidget):
     """Creates a custom table widget."""
-    def __init__(self, column_names, name="Table",
+    def __init__(self, points, name="Table",
                  textcolor="black", bgcolor="gray", parent=None, *args):
```

Comment:
It depends in the uses that we will give to it. If you think it will be more usefully the way it was we may change it back, but I will then have to convert Points to data array.